tools/osbuilder/node-builder/azure-linux/Makefile

```diff
@@ -0,0 +1,33 @@
+# Copyright (c) 2024 Microsoft Corporation
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+all: clean package uvm
+
+all-confpods: clean package-confpods uvm-confpods
+
+package:
+	./package_build.sh
+
+package-confpods:
+	CONF_PODS=yes ./package_build.sh
+
+uvm:
+	./uvm_build.sh
+
+uvm-confpods:
```

Comment:
We should make `uvm*` and `deploy*` depend on the build files, so that make knows not to do anything in case the build files haven't changed. Would probably help us figure out measurement issues too.

Reply:
Thanks! Let's revisit this in a future iteration.